### PR TITLE
Test for #14: middleware hangs when body-parser is incorrectly mounted.

### PR DESCRIPTION
**Pull request by dblock**
This was my attempt at reproducing #14, but I cannot get it to hang.

### CODE REVIEW

File: test/index.js [hunk 1]
```diff
@@ -125,3 +125,32 @@ test('fail invalid signature', function(t) {
...
+  var request = require('supertest');
+  request(server)
+    .post('/')
+    .send({ x: 1 })
+    .set('signaturecertchainurl', 'dummy')
+    .set('signature', 'aGVsbG8NCg==')
+    .end(function (err, res) {
+      t.equal(res.statusCode, 400);
+      t.deepEqual(res.body, {
+        "reason": "The raw request body is not available.",
```
Comment (tejashah88) [inline test/index.js]: When #19 is merged, the `reason` message should be [changed](https://github.com/alexa-js/alexa-verifier-middleware/pull/19/files#diff-168726dbe96b3ce427e7fedce31bb0bcR11) to `The raw request body has already been parsed.`

Comment (dblock) [reply test/index.js]: But that is cosmetic, the bug is that the request hangs not that the message is wrong.

Comment (tejashah88) [reply test/index.js]: I'm just saying that when #19 is merged, just remember to change the message.

File: test/index.js [hunk 2]
```diff
@@ -125,3 +125,32 @@ test('fail invalid signature', function(t) {
...
 });
+
+
+test('with express.js and body-parser incorrectly mounted', function(t) {
+  var express = require('express');
+  var app = express();
+  app.use(require('body-parser').json());
+  app.use(verifier);
+  var server = app.listen(3000);
+  app.get('/', function(req, res) {
```
Comment (tejashah88) [inline test/index.js]: `app.get` should be `app.post`

**Comment by dblock** [comment]
This is updated after #19 was merged. I think it's a useful test @tejashah88, even though it doesn't reproduce the hang, your call if you want to merge it.

**Comment by tejashah88** [comment]
I agree, this test is still useful. Merging...

**Pull request merged**